packages/protobuf/src/message.ts

```diff
@@ -141,8 +141,15 @@ export class Message<T extends Message<T> = AnyMessage> {
    * unpacked, and this is only possible with a type registry to look up the
    * message type.  As a result, attempting to serialize a message with this
    * type will throw an Error.
+   *
+   * Note that this function is private because users should not need to invoke
+   * it directly -- they can use JSON.stringify.  Additionally, if it is public
+   * it shows as part of the Message API to users which can be confusing when
+   * listed with the other toJson method
    */
-  toJSON(): JsonValue {
+  //  eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  //  @ts-ignore
+  private toJSON(): JsonValue {
```

Comment:
What do you think about making it `protected` to avoid the two linter comments?

Reply:
It's definitely worth making sure `protected` doesn't have unintended side-effects, but regarding subclassing, we clearly warn against doing that [here](https://github.com/bufbuild/protobuf-es/blob/b430d99fd8064b9073b2c817faa244257f5df5a2/packages/protobuf/src/message.ts#L37).